config.yml

```diff
@@ -5,13 +5,21 @@ defaults:
   # TaskCluster credentials
   taskcluster:
     rootUrl: !env TASKCLUSTER_ROOT_URL
+    queue: none
```

Comment:
It looks like this is unused (also, note that in yaml, `none` is a string -- I think you were thinking of `null`)